components/HomepageNews/HomepageNews.vue

```diff
@@ -67,9 +67,11 @@
               {{ item.fields.title }}
             </a>
           </h3>
-          <div class="sparc-card__detail">
-            <svg-icon name="icon-calendar" height="16" width="16" />
-            <p>{{ eventDate(item) }}</p>
+          <div class="sparc-card__detail" v-if="eventDate(item) && item.fields.location">
+            <template v-if="eventDate(item)">
```

Comment:
Can't see how the `v-if` is necessary given it also applies to the parent `div`.

Reply:
Good catch, the parent one should be or.

Reply:
It is possible to have the date or/and location.